package.json

```diff
@@ -40,7 +40,9 @@
     "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
     "test-no-cover": "repo-tools test run --cmd mocha -- test/*.js --no-timeouts",
-    "test": "repo-tools test run --cmd npm -- run cover"
+    "test": "repo-tools test run --cmd npm -- run cover",
```

Comment:
I wonder if it's possible to put the license check inside of the repo-tools command? @jmdobry ?

Reply:
I'm fine with making repo-tools support license checking but I think that is beyond this PR's scope :)

Any objections to committing this PR as is?

Reply:
Issue opened to track that request: https://github.com/GoogleCloudPlatform/nodejs-repo-tools/issues/94

Reply:
Thanks, @stephenplusplus !